interface/modules/custom_modules/oe-module-claimrev-connect/table.sql

```diff
@@ -0,0 +1,32 @@
+-- This table definition is loaded and then executed when the OpenEMR interface's install button is clicked.
+CREATE TABLE IF NOT EXISTS `mod_claimrev_eligibility`(
+    `id` INT(11)  PRIMARY KEY AUTO_INCREMENT NOT NULL
+    ,`pid` bigint(20)
+    ,`payer_responsibility` varchar(2)
+    ,`request_json` TEXT NULL
```

Comment:
For this text field and all below ones, remove the NULL:
```
,`request_json` TEXT
```

Reply:
done